components/snackbar/snackbar.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap");
+
+:root {
+  --snackbar-background: #1e293b;
+  --snackbar-retry-btn: #8a2be2;
+  --snackbar-retry-btn-bg: #363636;
+}
+.snackbar-buttons {
+  margin-top: 2.5rem;
+  display: flex;
+  justify-content: space-evenly;
+}
+.btn-snackbar {
+  color: var(--txt-grey-100);
+  font-weight: bold;
+  padding: 10px;
```

Comment:
suggestion/ refactor later : keep CSS units consistent ( if using rem use it across all the places)

reference: https://nekocalc.com/px-to-rem-converter

Reply:
Thanks for suggesting the resource , will surely look into it